### DISCUSSION TITLE
Yalan söyleyen ajanlar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#81] [Zeka Sorusu]
A,B,C,D,E şifre isimli 5 ajan sorguya çekildiklerinde şunları
söylüyorlar.
A : "C ve D yalan söylüyor"
B : "A ve E yalan söylüyor"
C : "B ve D yalan söylüyor"
D : "C ve E yalan söylüyor"
E : "A ve B yalan söylüyor"
Kimin -kesin olarak- yalan söylediğini bulunuz.

**Comment by HabilCcabbar** [comment]
Eğer A yalancı olsaydı A'nın dediğinden D'nin dürüst fakat C'nin dediğinden ise D'yi yalancı buluyoruz fakat A dürüst ise aynı durum tekrar oluyor. Hocam soru galiba hatalı.

**Reply by HabilCcabbar** [reply]
Hocam bence A onları sorgulayanların kafasını karıştırmak için böyle bir şey yapıyor.

**Reply by sonsuzus** [reply]
Soruda hata yok.

**Comment by HabilCcabbar** [comment]
[AjanlarÇelişki.xlsx](https://github.com/sonsuzus/sonsuzus/files/11994937/AjanlarCeliski.xlsx)
Hocam bütün ajanların ifadelerindeki çelişkiler.

**Reply by sonsuzus** [reply]
Dürüst kişiyi aramıyoruz ki... Kesin yalan söyleyeni arıyoruz.
